src/packages/fetch-engine/src/getLatestTag.ts

```diff
@@ -4,7 +4,7 @@ import { getDownloadUrl } from './util'
 import { platforms } from '@prisma/get-platform'
 import PQueue from 'p-queue'
 
-export async function getLatestAlphaTag(): Promise<any> {
+export async function getLatestTag(): Promise<any> {
```

Comment:
Intentional that this does not refer to `dev` any more?

Reply:
Yes, because it has nothing to do with `dev` or `alpha`, it just gets the latest tag.